### PR TITLE
Create getPointAttributes.js

### PR DESCRIPTION
**Pull request by adamwilliams87**
Example of how to get attributes for a context point in a meta script.

Please review and is there a better way to do this? I tested it live and it does work.

### CODE REVIEW

**Review by MertCingoz — APPROVED** [review]

File: mango-javascript/getPointAttributes.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var Common = Java.type('com.serotonin.m2m2.Common');
+var rt = Common.runtimeManager.getDataPoint(test.getDataPointWrapper().getId());
+if(rt.getAttributes()['UNRELIABLE']==true){
```
Comment (jazdw) [inline mango-javascript/getPointAttributes.js]: You don't need the `== true` here

**Review by jazdw — APPROVED** [review]
That looks about right, I suggested a small change

**Comment by adamwilliams87** [comment]
Thanks Jared. Terry helped me with it but he wasn't sure if there was a better way to do it using reflection. He had made the comment "inside wrapper use reflection to get field named point which is a DataPointRT".

If you think the way it was done is the best way then we'll go with it.

**Comment by terrypacker** [comment]
The only other thing that you would need to consider is if the data point is disabled.  It will still be in the context but the resulting rt would be null.  So a safer way would be this:

```
var rt = Common.runtimeManager.getDataPoint(test.getDataPointWrapper().getId());
if(rt != null && rt.getAttributes()['UNRELIABLE']==true) {
    // do stuff here
}
```

**Comment by jazdw** [comment]
> It will still be in the context but the resulting rt would be null. 

Good catch didn't spot that. Terry's code could be simplified to this in JavaScript:

```
var rt = Common.runtimeManager.getDataPoint(test.getDataPointWrapper().getId());
if (rt && rt.getAttributes()['UNRELIABLE']) {
    // point is running, and has unreliable attribute
}
```

**Comment by jazdw** [comment]
@adamwilliams87 what is happening with this?

**Pull request merged**